packages/peregrine/lib/talons/SignIn/useSignIn.js

```diff
@@ -29,6 +32,12 @@ export const useSignIn = props => {
         errors.push(getDetailsError);
     }
 
+    useEffect(() => {
+        if (getCustomerData) {
```

Comment:
This will rerun if the data object changes at all. Is that what we want?

Reply:
Yes, though that use case should be impossible, given the component is unmounted right after this.